src/systems/filecoin_mining/storage_mining/storage_miner_actor.go

```diff
@@ -724,7 +729,9 @@ func (a *StorageMinerActorCode_I) ProveCommitSector(rt Runtime, info sector.Sect
 	isSealVerificationCorrect := st._isSealVerificationCorrect(rt, onChainInfo)
 	if !isSealVerificationCorrect {
 		// TODO: determine proper error here and error-handling machinery
```

Comment:
are we ready to remove these TODOs on error handling?

Reply:
Yes, I think so. I tried to make as few changes to existing code as possible, but can definitely take another pass and see which of these types of comments can now be removed.